superdesk/publish/subscriber_token.py

```diff
@@ -31,5 +32,6 @@ class SubscriberTokenService(superdesk.Service):
     def create(self, docs, **kwargs):
         for doc in docs:
             doc['_id'] = get_random_token()
-            doc.setdefault('expiry', utcnow() + timedelta(days=7))
+            if not doc.get('never_expire'):
+                doc.setdefault('expiry', utcnow() + timedelta(days=7))
```

Comment:
can we set `never_expire` and `expiry` together?

Reply:
Nope, If we have `never_expire` as null, then only we should set the expiry otherwise there is no need to set it.

Reply:
it might be easier for client if we set expiry readonly and rather send some `expires_after` number which would be the number of days, if not set it would never expire.

Reply:
maybe rather `expiry_days` to make the name explicit